src/consumer/index.js

```diff
@@ -144,7 +144,7 @@ module.exports = ({
    * @param {boolean} [fromBeginning=false]
    * @return {Promise}
    */
-  const subscribe = async ({ topic, fromBeginning = false }) => {
+  const subscribe = async ({ topic, fromBeginning = false, fromTimestamp = null }) => {
```

Comment:
I would like to have a way to test this as well. Currently only the admin functionality is being tested.

Reply:
Removed functionalities in consumer part.